hack/deploy-karmada.sh

```diff
@@ -197,9 +199,12 @@ then
   recover_kubeconfig
   exit 1
 fi
-util::fill_cabundle "${ROOT_CA_FILE}" "${REPO_ROOT}/charts/_crds/patches/webhook_in_resourcebindings.yaml"
-util::fill_cabundle "${ROOT_CA_FILE}" "${REPO_ROOT}/charts/_crds/patches/webhook_in_clusterresourcebindings.yaml"
-installCRDs
+
+TEMP_PATH_CRDS=$(mktemp -d)
```

Comment:
How this temp path is cleaned?

Reply:
Now Fixed. Clean the directory after installation.